DTS-STN/dts-rmp
# 116 add comment existing eng

# Description

[#116 Add comment to existing engagement (client side only)]
https://taiga.dts-stn.com/project/pond-knowlege-management-portal/us/116?milestone=20

Added capability to add comments for an engagement. (only client side)

# Story Acceptance Criteria

Respecting both AC in taiga. 

Get the acceptance criteria for the story relating to your task and put all of it here.

## Test Instructions

1. Select an engagement.
2. scroll down to the comment section
3. add a comment and see it added to the list


## Help Requested

The box to view all comments might be a bit small. Let me know if I should make it bigger.
(User and date are hardcoded strings)

## Checklist
